TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/bot/BotActivity.java

```diff
@@ -174,10 +174,15 @@ protected void onStop() {
             unbindService(serviceConnection);
             boundToMovementService = false;
         }
-        presenter.stopPresenting();
         super.onStop();
     }
 
+    @Override
+    protected void onDestroy() {
```

Comment:
So basically we would still close the connection when activity is destroyed ?

So having this is mind when device would low run in memory and decide to kill the activities that are in the background , then the service would be destroyed as well right ?